homeassistant/components/lookin/media_player.py

```diff
@@ -177,39 +144,3 @@ def _update_from_status(self, status: str) -> None:
 
         self._attr_state = STATE_STANDBY if state == "1" else STATE_ON
```

Comment:
I found this didn't work as expected as this looks like a logic reversal

`        self._attr_state = STATE_STANDBY if state == "1" else STATE_ON`

I'll address it in another PR though since the fix should probably be backported

Reply:
https://github.com/home-assistant/core/pull/64136